BUILD.bazel

```diff
@@ -2744,3 +2743,34 @@ genrule(
     """,
     local = 1,
 )
+
+genrule(
+    name = "compile_py_proto",
+    srcs = [
+        "@io_opencensus_proto//opencensus/proto/metrics/v1:metrics.proto",
+        "@io_opencensus_proto//opencensus/proto/resource/v1:resource.proto",
+    ] + glob([
+        "src/ray/protobuf/**/*.proto",
```

Comment:
I feel not all proto is needed actually. For example, not all proto is compiled with py.

Reply:
Yea, I just keep it simple to compile everything, it should be no harm.